custom_components/oasis_mini/button.py

```
@@ -13,68 +13,103 @@
+    """
+    Play a random track on the given Oasis device.
+    
+    Selects a track at random from the available TRACKS and attempts to add it to the device's queue and play it. Raises HomeAssistantError if adding the track times out.
+    
+    Parameters:
+        device: The Oasis device on which to play the track.
+    
+    Raises:
+        HomeAssistantError: If adding the selected track to the device's queue times out.
+    """
     track = random.choice(list(TRACKS))
```

Ruff (0.14.5)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)